02-configuration.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: page
+title: Version Control with Mercurial
+subtitle: Configuring Mercurial
+---
+> ## Learning Objective {.objectives}
+>
+> * Explain what configuration steps are required the first time Mercurial
+>   is used on a computer.
+
+When we use Mercurial on a computer for the first time,
+we need to configure a few things.
+The command
+
+~~~ {.bash}
+$ hg config --edit
```

Comment:
Good catch! Looks like `hg config --edit` was introduced in Mercurial v3.0 (2014-05-01).

Yes, the previous version had direct editing of the config file. `hg config --edit` avoids the diversion of where the config file is on different platforms and what it is called. Would it work to put the direct editing approach in a callout to be used in the cases where `hg config --edit` fails?